service/src/main/kotlin/fi/espoo/evaka/shared/security/Action.kt

```diff
@@ -196,7 +198,9 @@ sealed interface Action {
         UPDATE_CHILD_RECIPIENT(SERVICE_WORKER, UNIT_SUPERVISOR),
 
         CREATE_VASU_DOCUMENT(UNIT_SUPERVISOR, SPECIAL_EDUCATION_TEACHER, GROUP_STAFF),
-        READ_VASU_DOCUMENT(UNIT_SUPERVISOR, SPECIAL_EDUCATION_TEACHER, GROUP_STAFF);
+        READ_VASU_DOCUMENT(UNIT_SUPERVISOR, SPECIAL_EDUCATION_TEACHER, GROUP_STAFF),
+
+        READ_PEDAGOGICAL_DOCUMENT(UNIT_SUPERVISOR, SPECIAL_EDUCATION_TEACHER);
```

Comment:
I wonder if we should have CREATE_PEDAGOGICAL_DOCUMENT here, under Child actions